docs/_panels/jenkins-nodes.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Jenkins Nodes
+description: Information about Jenkins Nodes
+author: Bitergia
+screenshot: sigils/jenkins-nodes.png
+created_at: 2020-03-11
+grimoirelab_version: 0.2.38
+layout: panel
+---
+
+This panel provides an overview of the nodes used in the Jenkins instances.
```

Comment:
I doubt whether `used by the Jenkins instances` would be more accurate in this context. Not a big deal, though.